python/pip_install/tools/wheel_installer/wheel.py

```diff
@@ -80,6 +82,39 @@ class Platform:
     os: OS
     arch: Optional[Arch] = None
 
+    @classmethod
+    def all(cls, want_os: Optional[OS] = None) -> List["Platform"]:
+        return [
+            cls(os=os, arch=arch)
+            for os in OS
+            for arch in Arch
+            if not want_os or want_os == os
+        ]
+
+    @classmethod
+    def host(cls) -> "Platform":
+        return [
+            cls(os=OS[sys.platform.lower()], arch=Arch[platform.machine()])
+            for os in OS
+            for arch in Arch
+        ]
```

Comment:
This looks wrong? "For every OS and arch combination, return the host os/arch" ? I think you meant to not have a list comp here?

Reply:
Yeah, this got fixed very recently.